docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -116,6 +116,25 @@ Custom yaml settings are silently ignored by {fleet}.
 Settings with input blocks, such as Max agents are still effective.
 ====
 
+[[known-issue-issue-2249-8.6.2]]
+.{agent} does not load fleet configuration after upgrade from unencrypted config
```

Comment:
You need to specify which versions have unencrypted configuration. If you are a user how do you know this applies to you?

Reply:
Save them from having to read the full summary, which is collapsed by default